fastify/fastify
# fix: citgm

Following https://github.com/nodejs/citgm/pull/973

Tested locally on node 21 nightly

test files:
- `custom-parser.1.test.js`
- `listen.test.js`
- `plugin.test.js`
- `route.test.js`
- `stream.test.js`

Have been splitted because they contained too many tests, that hit timeout on slow citgm machines

Note before merge:

- remove debugging `console.log`s
- restore `npm test` as it was
- make sure `citgm` script is been used on citgm, must be in `lookup.json` file or citgm settings

Note to future me: run it as `citgm git+https://github.com/simone-sanfratello/fastify#fix/citgm -l lookup.json`

lookup.json
```json
{
  "useGitClone": true,
  "scripts": ["citgm"]
}
```
